src/components/edit-listing-schedule/ListingSchedule/edit-listing-schedule.js

```diff
@@ -0,0 +1,107 @@
+import Button from 'components/shared/Button';
+import DatePicker from 'react-datepicker';
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
+
+export const DateInput = styled(DatePicker)`
+  background-color: transparent;
+  border-radius: 1.75rem;
+  border: none;
+  color: ${colors.mineshaft};
+  flex: 1;
+  font-size: 0.75rem;
+  font-weight: 400;
+  outline: none;
+  padding: 1rem 1rem 1rem 2rem;
+`;
+
+export const DateInputWrapper = styled.div`
+  align-items: center;
+  background-color: ${colors.white};
+  border-radius: 2.18rem;
+  border: 0.06rem ${colors.mineShaft}16 solid;
+  display: flex;
+  margin: 1.3rem 3rem;
+  max-width: 90%;
+  position: relative;
+
+  input {
+    width: 100%;
+  }
+
+  /* while you picking a date color */
+  .react-datepicker__day--in-selecting-range:not(.react-datepicker__day--in-range) {
+    background-color: ${colors.green}99;
```

Comment:
```suggestion
    background-color: ${colors.green};
    opacity: 0.5;
```
Please don't attach string parts to the variables. The point is to keep the css values comprehensive and clear.